content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Enabling HTTPS with InfluxDB
+description: >
+  Enable HTTPS and Transport Security Layer (TLS) secure communication between clients and your InfluxDB servers.
```

Comment:
Enable Transport Layer Security (TLS) and use HTTPS to secure communication between clients and InfluxDB.

Reply:
Actually, in these descriptions, we probably don't need to worry about provided expended initialisms. These are used for the HTML page descriptions and what get scraped by search engines. They'll know what to do with just "TLS".